### ISSUE TITLE
NEW FEATURE: Include

### ISSUE THREAD

**Issue by Jara92** [SamKirkland/FTP-Deploy-Action#201]
It would be very perfect to implement "include" feature. I miss it when I need to sync only few folders. Exclude is not very handy for that.

**Comment by shawninder** [comment]
Indeed, many times I find whitelisting easier than blacklisting…

Using both exclude and include together can also be powerful:
```
exclude:
  - .*
include:
  - .exception
```

Right now I'm really not sure how to do this or if the following might work:
```
exclude:
  - .*
  - !.exception
```
Wish me luck!

**Comment by favalonx** [comment]
Yeah. There are usually a lot of configs files, folders, and dependencies that are not necessary for the production. It would be great if it only uploads listed files and folders mentioned in `includes` .
